libs/ui-toolkit/src/components/copy-with-tooltip/copy-with-tooltip.tsx

```diff
@@ -15,7 +15,7 @@ export function CopyWithTooltip({ children, text }: CopyWithTooltipProps) {
 
   return (
     <CopyToClipboard text={text} onCopy={() => setCopied(true)}>
-      {/* Needs this wrapping div as tooltip component interfers with element used to capture click for copy */}
+      {/* Needs this wrapping div as tooltip component interfere with element used to capture click for copy */}
```

Comment:
need the 's' for correct grammar

```suggestion
      {/* Needs this wrapping div as tooltip component interferes with element used to capture click for copy */}
```